docs/project/scos-migration-faq.md

```diff
@@ -1,10 +1,50 @@
-# FAQ - OKD Components based on SCOS
-
+# OKD SCOS 4.16 Frequently Asked Questions
+ 
 <!--cSpell:ignore SCOS -->
 
 This FAQ document relates to the builds of OKD that use CentOS Stream CoreOS (SCOS) after work done to stop using RHEL as the base for cluster components, as well as the upgrade path from pre-4.16 OKD FCOS/SCOS installations.
 
-## Secure Boot
-Secure Boot is currently not supported on CentOS Stream CoreOS (SCOS). This is tracked in [RHEL-4391](https://issues.redhat.com/browse/RHEL-4391) and [rhboot/shim-review#399](https://github.com/rhboot/shim-review/issues/399).
+- [When will OKD SCOS be released?](https://hackmd.io/JkM3HQC7Q_2Y8hjMiXytyw?both#When-will-OKD-SCOS-be-released)
```

Comment:
I think the links to the hackmd should actually be page links to below. So for example instead of:
https://hackmd.io/JkM3HQC7Q_2Y8hjMiXytyw?both#When-will-OKD-SCOS-be-released
should be:
https://okd.io/docs/project/scos-migration-faq#When-will-OKD-SCOS-be-released